extension/src/test/unit/gradleTasks.test.ts

```diff
@@ -235,7 +235,6 @@ describe(getSuiteName('Gradle tasks'), () => {
             groupItem.label,
             mockTaskDefinition1ForFolder1.group
           );
-          assert.strictEqual(groupItem.iconPath, vscode.ThemeIcon.Folder);
```

Comment:
since we changed the themeicon in https://github.com/microsoft/vscode-gradle/pull/887/files#diff-183471d1a24e3c81ad33c4c977c62b13d94770466ca974bb60b5efcf3d6778d2 , IMO there is no way to check the icon so I simply remove it.